cli/CMakeLists.txt

```diff
@@ -44,6 +44,14 @@ set_tests_properties(test_short_version_switch PROPERTIES
 add_test(multiple_files_on_command_line ${EDITORCONFIG_CMD} -f cli.in
     "${CMAKE_CURRENT_SOURCE_DIR}/file1.c"
     "${CMAKE_CURRENT_SOURCE_DIR}/file2.cpp")
+
+make_regex_of_permutations(     # each pair of lines must be in that order,
```

Comment:
The comment here does not seem clearly to me. What does it mean?

Reply:
We are making a `multi_files_regex` that will accept either

    [...file1.c]
    key1=value1
    [...file2.cpp]
    key2=value2

or

    [...file2.cpp]
    key2=value2
    [...file1.c]
    key1=value1

Each of the `EXPECTED` regexes includes two lines, so to pass the test two consecutive lines have to match that regex.  However, the `multi_files_regex` will accept the two lines for `file1.c` first, or the two lines for `file2.cpp` first.

I added the comment to make the point that the `EXPECTED` regexes were not single lines, as they are in most other tests.